lib/chai/interface/assert.js

```diff
@@ -1213,173 +1178,260 @@ module.exports = function (chai, util) {
   };
 
   /**
-   * ### .nestedPropertyVal(object, property, value, [message])
+   * ### .ownProperty(object, property, [message])
    *
-   * Asserts that `object` has a property named by `property` with value given
-   * by `value`. `property` can use dot- and bracket-notation for nested
-   * reference. Uses a strict equality check (===).
+   * Asserts that `object` has a direct property named by `property`.
```

Comment:
I think that only saying "direct property" isn't enough to make this explanation clear, maybe we could add `(non-inherited)` after the word `direct` just to make sure we're being clear.

I don't think this is needed in the other methods where we mention inherited properties, since that contrast already explains everything by itself.